ha/monitor/k8s/const.py

```diff
@@ -21,6 +21,8 @@ class K8SEventsConst:
     RAW_OBJECT = 'raw_object'
     METADATA = 'metadata'
     NAME = 'name'
+    LABELS ='labels'
+    MACHINEID = 'cortx.io/machine-id'
```

Comment:
Can this be split into 2 constsnts ? cortx.io and machine-id ?

Reply:
As discussed this split is not required as this is the single key.
also as discussed with @osowski this is constant so it will be used as it is constant.
so adding a comment as discussed.

Reply:
done

Reply:
done